scss/_base_icon-definitions.scss

```diff
@@ -506,3 +506,32 @@ $social-icon-size: map-get($icon-sizes, heading-icon--small);
 @mixin vf-icon-warning-grey($color) {
   background-image: url("data:image/svg+xml,%3Csvg width='16' height='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M8.734.289a1 1 0 01.429.429l6.631 12.823A1 1 0 0114.906 15H1.643a1 1 0 01-.888-1.46L7.386.719a1 1 0 011.348-.43zm-.46 1.976L2.464 13.5h11.62L8.274 2.265zm.5 8.022a.5.5 0 01.492.41l.008.09v1a.5.5 0 01-.41.492l-.09.008h-1a.5.5 0 01-.491-.41l-.009-.09v-1a.5.5 0 01.41-.492l.09-.008h1zm.5-4.68V8.98h-2V5.608h2z' fill='#{vf-url-friendly-color($color)}'  fill-rule='nonzero'/%3E%3C/svg%3E");
 }
+
+@mixin vf-icon-linux-prompt($color) {
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='16' height='16'%3E%3Cpath fill='#{vf-url-friendly-color($color)}'  d='M8.608 13.14V11.3c.864-.107 1.504-.379 1.92-.816.416-.437.624-.97.624-1.6 0-.416-.077-.773-.232-1.072a2.464 2.464 0 00-.616-.768 3.912 3.912 0 00-.896-.552 18.991 18.991 0 00-1.056-.44 20.136 20.136 0 01-.776-.304 3.321 3.321 0 01-.6-.32 1.327 1.327 0 01-.392-.4 1.062 1.062 0 01-.136-.56c0-.405.152-.715.456-.928.304-.213.744-.32 1.32-.32.523 0 .955.045 1.296.136.341.09.624.179.848.264l.272-1.12a5.789 5.789 0 00-.856-.248A6.99 6.99 0 008.608 2.1V.5H7.424v1.648c-.768.117-1.355.39-1.76.816-.405.427-.608.981-.608 1.664 0 .384.07.707.208.968.139.261.317.485.536.672.219.187.47.344.752.472.283.128.568.245.856.352.299.107.587.216.864.328.277.112.525.24.744.384.219.144.392.31.52.496.128.187.192.413.192.68 0 .181-.032.35-.096.504a.954.954 0 01-.32.4c-.15.112-.355.197-.616.256a4.506 4.506 0 01-.968.088c-.597 0-1.093-.061-1.488-.184a7.53 7.53 0 01-1.024-.392l-.352 1.104c.203.117.512.24.928.368.416.128.96.203 1.632.224v1.792h1.184z'/%3E%3C/svg%3E");
+}
+
+%icon {
+  @extend %vf-hide-text;
+  @include vf-icon-size($default-icon-size);
+  $vertical-offset: 0.5px;
+  background: {
+    position: center;
+    repeat: no-repeat;
+    size: contain;
+  }
+
+  display: inline-block;
+  font-size: inherit; // allow icons to match size of parent text element, when set in em
+  margin: 0;
+  padding: 0;
+  position: relative;
+  vertical-align: calc(#{$vertical-offset} + #{0.5 * $cap-height} - #{0.5 * $default-icon-size});
+}
+
+%social-icon {
+  @extend %vf-hide-text;
+  @include vf-icon-size($social-icon-size);
```

Comment:
Just making sure - is `vf-icon-size` defined in this file or in icons pattern?

Reply:
@bartaz yep, it's the first mixin in this file.